source-map-support.js

```diff
@@ -84,7 +84,7 @@ function retrieveSourceMapURL(source) {
 
   // Get the URL of the source map
   fileData = retrieveFile(source);
-  var re = /\/\/[#@]\s*sourceMappingURL=([^'"]+)\s*$/mg;
+  var re = /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
```

Comment:
For the syntax regarding {1} see the Quantifiers table at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp